lib/data/model/genres_model.dart

```diff
@@ -0,0 +1,15 @@
+import '../../domain/entity/genres_entity.dart';
+
+class GenresModel extends GenresEntity {
```

Comment:
I'd rename `GenresModel` and `GenresEntity` to singular instead of plural, as they represent a single `Genre` entity

Reply:
Done!